frontend/src/pages/CardPage/components/Path/Path.jsx

```diff
@@ -1,11 +1,37 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import styles from './Path.module.scss';
 import ROUTES from '../../../../constants/routers';
-import { Link } from 'react-router-dom';
+import { Link, useParams } from 'react-router-dom';
 import { ICONS } from '../../../../components/ui-kit/icons';
+import { GetCategory } from '../../../../services/catalog/categoryAccess.service';
 
 const Path = React.memo(({ currentPageData, currentPageType }) => {
+  const [parents, setParents] = useState([]);
+  const { categoryId } = useParams();
+
+  useEffect(() => {
+    if (currentPageType !== 'section') {
+      const fetchCategoryAndSubcategory = async (categoryId) => {
```

Comment:
It would be better to move function outside useEffect

Reply:
can you explain why?